core/arch/proc_api.h

```diff
@@ -347,8 +399,9 @@ proc_has_feature(feature_bit_t feature);
 
 DR_API
 /**
- * Returns all 4 32-bit feature values.  Use proc_has_feature to test
- * for specific features.
+ * Returns all 4 32-bit feature values on X86 and architectural feature
+ * registers' values on AArch64. Use proc_has_feature to test for specific
```

Comment:
I know we don't currently have systematic x86 tests for features (I just see a few uses in the clean call test) but it would be good to add something: could we add some kind of sanity test that asking about features works?

Reply:
Do you mean client tests?

Currently the LOG_FEATURE macro puts out text in the log file for debug builds which exercises the register access and feature bits interpretation code.

Reply:
Right, a client using the API routines seems like a good addition.

Reply:
Added client-interface/features.dll.c unit test.